### PR TITLE
Star

### PR DESCRIPTION
**Pull request by stu2**
Hi Phil, 

Just a few modifications to get STAR running with your recent overhaul of CF. Previous pull request didn't fix star output for paired-end mode.

Cheers, 

Stuart

### CODE REVIEW

**Comment by ewels** [comment]
Thanks @stu2! As you keep pushing commits the PR is updated, so there's no need to close and re-open.. (in fact, the closed PR looks identical to this one now)

File: modules/star.cfmod.pl [hunk 1]
```diff
@@ -45,21 +45,22 @@
 # --mem. Return the required memory allocation.
 if($required_mem){
     if(!$run_fn){
-        print '10G';
+        print '32000000000';
```
Comment (ewels) [inline modules/star.cfmod.pl]: It doesn't matter, but just so you know - this is parsed by the core `cf` program into bytes, so for increased code readability, outputting `32G` here is fine..

**Comment by stu2** [comment]
Thanks, good to know.

File: modules/star.cfmod.pl [hunk 2]
```diff
@@ -45,21 +45,22 @@
...
         exit;
     } else {
         # Parse the run file
         my ($starting_files, $config) = CF::Helpers::parse_runfile($run_fn);
     	# estimate memory based on genome size
-        my $minmem = 32000111000;
+        my $minmem = 32000000000;
```
Comment (ewels) [inline modules/star.cfmod.pl]: Hmm, I'm not sure - will it still work without that `0.0001GB` of memory..? :laughing:

**Comment by ewels** [comment]
Great, all looks good.. Merging. Thanks!

File: modules/star.cfmod.pl (hunk 2 above)
Comment (stu2) [inline modules/star.cfmod.pl]: Well it frees up enough to play Pong while you wait.

Comment (ewels) [inline modules/star.cfmod.pl]: pong

**Pull request merged**